rasa/core/policies/ted_policy.py

```diff
@@ -871,16 +954,121 @@ def load(
             for tag_spec in entity_tag_specs
         ]
 
-        model_data_example = RasaModelData(
-            label_key=LABEL_KEY, label_sub_key=LABEL_SUB_KEY, data=loaded_data
+        return {
+            "tf_model_file": tf_model_file,
+            "loaded_data": loaded_data,
+            "fake_features": fake_features,
+            "label_data": label_data,
+            "meta": meta,
+            "priority": priority,
+            "entity_tag_specs": entity_tag_specs,
+        }
+
+    @classmethod
+    def load(
+        cls,
+        path: Union[Text, Path],
+        should_finetune: bool = False,
+        epoch_override: int = defaults[EPOCHS],
+        **kwargs: Any,
+    ) -> Optional["TEDPolicy"]:
```

Comment:
The interface expects a `Policy` to be returned (see [here](https://github.com/RasaHQ/rasa/blob/8db9030bf7ec457e9306c95cbf7fbe811771d05f/rasa/core/policies/policy.py#L374)) and not `Optional`. An exception seems to be the better option than `logger.error` hence.